CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+0.4.2 / 2014-02-..
```

Comment:
2024 :)

Reply:
Date needs to be updated with a day, but good observation 👍